packages/kununu-header/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../../.babelrc"
```

Comment:
I always thought if you have a .babelrc in the root any sub packages/dirs that use babel will look for it in parent folders?

Reply:
It needs the `--root-mode upward` option, now all files are gone :)